src/odemis/acq/stitching/__init__.py

```diff
@@ -37,57 +38,57 @@ def register(tiles, method=REGISTER_SHIFT):
         tiles (list of DataArray of shape YX or tuples of DataArrays): The tiles as passed, but with updated 
         MD_POS metadata
     """
-    
+
     if method == REGISTER_SHIFT:
-        registrar = ShiftRegistrar()        
+        registrar = ShiftRegistrar()
     elif method == REGISTER_IDENTITY:
         registrar = IdentityRegistrar()
     else:
-        raise ValueError("Invalid registrar %s" %(method)) 
+        raise ValueError("Invalid registrar %s" % (method))
```

Comment:
Note that `(x)` in Python is _not_ a tuple, it's the same as `x`. To get a tuple of one element, you have to type `(x,)`. That said, in this very case, it will still work, because % accepts either one element or a tuple of elements. It's not very recommended to pass just one element, because if `method` happens to be a tuple (or list), % will try to format each item and it will fail.